src/libsyntax/diagnostic_list.rs

```diff
@@ -16,31 +16,374 @@
+```
+#[cfg(not(target_os = "linux"))] // ok!
+pub fn something() {}
+
+fn main() {}
+```
+
+For more information about the cfg attribute, take a look here:
+https://doc.rust-lang.org/reference.html#conditional-compilation
+"##,
+
+E0538: r##"
+The same item was used more than once on a single element.
+
+Erroneous code example:
+
+```compile_fail
+#[unstable(feature = "some_feature", issue = "1")]
+#[unstable(feature = "another_feature", issue = "1")]
+// error: multiple stability items
+pub fn something() {}
+```
+
+A same item cannot be used multiple times on a single element. Example:
+
+```ignore
+#[unstable(feature = "another_feature", issue = "1")] // ok!
+pub fn something() {}
+```
+"##,
+
+E0539: r##"
+A meta item doesn't have a value.
+
+Erroneous code example:
+
+```compile_fail
+#[stable(feature)] // error: incorrect meta item
+pub fn something() {}
+```
+
+Meta items need a value:
+
+```ignore
+#[stable(feature = "some_feature")] // error: incorrect meta item
+pub fn something() {}
+```
+"##,
+
+E0540: r##"
+rustc_deprecated attribute was used more than once on an element.
+
+Erroneous code example:
+
+```compile_fail
+#[rustc_deprecated(since = "1.3.0", reason = "some reason")]
+#[rustc_deprecated(since = "1.4.0", reason = "no reason")]
+// error: multiple rustc_deprecated attributes
+pub fn something() {}
+```
+
+For obvious reason, an element can only be deprecated once. Example:
+
+```ignore
+#[rustc_deprecated(since = "1.3.0", reason = "some reason")] // ok!
+pub fn something() {}
+```
+"##,
+
+E0541: r##"
+An unknown meta item was used on rustc_deprecated attribute.
+
+Erroneous code example:
+
+```compile_fail
+#[rustc_deprecated(since = "1.3.0", reason = "some reason", some = "unknown")]
+// error: unknown meta item
+pub fn something() {}
+```
+
+The rustc_deprecated attribute only recognizes the two following meta items:
+
+ * since
+ * reason
+
+Example:
+
+```ignore
+#[rustc_deprecated(since = "1.3.0", reason = "some reason")] // ok!
+pub fn something() {}
+```
+"##,
+
+E0542: r##"
+The since meta item wasn't used in rustc_deprecated attribute.
+
+Erroneous code example:
+
+```compile_fail
+#[rustc_deprecated(reason = "some reason")] // error: missing 'since'
+pub fn something() {}
+```
+
+The two following meta items must be given to the rustc_deprecated attribute:
+
+ * since
+ * reason
+
+Example:
+
+```ignore
+#[rustc_deprecated(since = "1.3.0", reason = "some reason")] // ok!
+pub fn something() {}
+```
+"##,
+
+E0543: r##"
+The reason meta item wasn't used in rustc_deprecated attribute.
+
+Erroneous code example:
+
+```compile_fail
+#[rustc_deprecated(since = "1.3.0")] // error: missing 'reason'
+pub fn something() {}
+```
+
+The two following meta items must be given to the rustc_deprecated attribute:
+
+ * since
+ * reason
+
+Example:
+
+```ignore
+#[rustc_deprecated(since = "1.3.0", reason = "some reason")] // ok!
+pub fn something() {}
+```
+"##,
+
+E0544: r##"
+Multiple stability levels were given on an element.
```

Comment:
Conflicting stability levels applied to an item.

Its not an error to have multiple stability annotations on an item (stable + deprecated is a thing, for example)